.github/pull_request_template.md

```diff
@@ -1,14 +1,94 @@
-## Description
-Provide a description of what this pull request is adding, updating, extending, or removing. If this request is addressing an issue, please link to that issue.
+# PR Title
```

Comment:
Could we add another section right before this one that says: "What Jira ticket is related to this PR?", so that we remember to link it to the ticket

Reply:
sure! done.